integration-tests/smoke/keeper_test.go

```diff
@@ -1103,71 +1058,43 @@ func TestKeeperUpdateCheckData(t *testing.T) {
 				"Expected perform data checker counter to be greater than 5, but got %d", counter.Int64())
 			l.Info().Int64("Upkeep perform data checker", counter.Int64()).Msg("Number of upkeeps performed")
 		}
+		return nil
 	}, "3m", "1s").Should(gomega.Succeed())
 }
 
-func setupKeeperTest(
-	t *testing.T,
-	testName string,
-) (
-	chainClient blockchain.EVMClient,
-	chainlinkNodes []*client.ChainlinkK8sClient,
-	contractDeployer contracts.ContractDeployer,
-	linkToken contracts.LinkToken,
-	onlyStartRunner bool,
+func setupKeeperTest(t *testing.T) (
+	*blockchain.EthereumClient,
+	[]*client.ChainlinkClient,
+	contracts.ContractDeployer,
+	contracts.LinkToken,
+	*test_env.CLClusterTestEnv,
 ) {
-	network := networks.SelectedNetwork
-	evmConfig := eth.New(nil)
-	if !network.Simulated {
-		evmConfig = eth.New(&eth.Props{
-			NetworkName: network.Name,
-			Simulated:   network.Simulated,
-			WsURLs:      network.URLs,
-		})
-	}
-
-	chainlinkChart, err := chainlink.NewDeployment(5, map[string]interface{}{
-		"toml": client.AddNetworksConfig(keeperBaseTOML, network),
-	})
-	require.NoError(t, err, "Error creating chainlink deployment")
-
-	networkName := strings.ReplaceAll(strings.ToLower(network.Name), " ", "-")
-	testEnvironment := environment.New(
-		&environment.Config{
-			NamespacePrefix: fmt.Sprintf("smoke-keeper-%s-%s", testName, networkName),
-			Test:            t,
-		}).
-		AddHelm(mockservercfg.New(nil)).
-		AddHelm(mockserver.New(nil)).
-		AddHelm(evmConfig).
-		AddHelmCharts(chainlinkChart)
-	err = testEnvironment.Run()
+	// TODO: Add ability to use non simulated network
```

Comment:
@ilija42 is currently working on enabling both load and smoke tests to run on multiple networks. I think you can skip implementing it in your branch @tateexon https://github.com/smartcontractkit/chainlink/pull/10178

Reply:
You were probably thinking about @iljapavlovs  :D

Reply:
Yep, thanks! :D